src/test/java/core/basesyntax/strategy/AddOperationHandlerTest.java

```diff
@@ -17,16 +17,16 @@
     @Test
     public void test_AddingFruitsToStorage_OK() {
         Storage.storage.put(apple, 10);
-        int expected = 10;
-        int actual = addHandler.apply(new Transaction(Operation.P, apple, 0));
+        int expected = 20;
+        int actual = addHandler.apply(new Transaction(Operation.S, apple, 10));
         assertEquals(expected, actual);
     }
 
     @Test
     public void test_AddingWrongData_OK() {
         Storage.storage.put(apple, 10);
         int expected = 0;
-        int actual = addHandler.apply(new Transaction(Operation.P, apple, 0));
+        int actual = addHandler.apply(new Transaction(Operation.S, apple, 0));
         assertNotEquals(expected, actual);
     }
```

Comment:
>this test does not look good
let's better check with return and supply operations separately

what about this comment?